README.md

```diff
@@ -100,8 +100,8 @@ Plugin to manage github action files.
     <tr>
       <td>next version</td>
       <td>= 0.22.0</td>
-      <td>= 0.7.1</td>
-      <td>= 0.3.0</td>
+      <td>= 0.8.0</td>
+      <td>= 0.3.1</td>
```

Comment:
update plugins' version but not plugin-core ? (0.22.0 -> 0.23.0)

Reply:
done